stepconf/stepconf.go

```diff
@@ -59,6 +59,18 @@ func Print(config interface{}) {
 	fmt.Printf(toString(config))
 }
 
+func valueString(v reflect.Value) string {
+	if v.Kind() != reflect.Ptr {
+		return fmt.Sprintf("%v", v.Interface())
+	}
+
+	if !v.IsNil() {
+		return fmt.Sprintf("%v", v.Elem().Interface())
+	}
+
+	return ""
```

Comment:
Should not "nil" be printed here to see the difference between empty string and nil?

Reply:
It is meaningless for the users, and except string type it is meaning for us. I think. I wouldn't print nil for the users.

Reply:
It is also useful as a debug log, so could print "unspecified" or similar.